src/System.Memory/tests/Performance/Perf.Utf8Parser.cs

```diff
@@ -201,7 +710,29 @@ private static void ParserUInt64(string text)
                 {
                     for (int i = 0; i < Benchmark.InnerIterationCount; i++)
                     {
-                        Utf8Parser.TryParse(utf8ByteSpan, out ulong value, out int bytesConsumed);
+                        Utf8Parser.TryParse(utf8ByteSpan, out ushort value, out int bytesConsumed);
+                        TestHelpers.DoNotIgnore(value, bytesConsumed);
+                    }
+                }
+            }
+        }
+
+        [Benchmark(InnerIterationCount = InnerCount)]
+        [InlineData("12837467")] // standard parse
+        [InlineData("-9223372036854775808")] // min value
+        [InlineData("9223372036854775807")] // max value
+        private static void ByteSpanToInt64(string text)
```

Comment:
There is no string baseline for this test. Also add the zero-padded and non-number prefix test cases similar to UInt64: https://github.com/dotnet/corefx/pull/32026/files#diff-7f5a11a35cf8403d05b6acd67bc83b28R110